package.json

```diff
@@ -151,7 +151,17 @@
         ],
         "url": "https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "flatpak-vscode",
+      "properties": {
+        "flatpak-vscode.extensionsIntegration": {
+          "type": "boolean",
+          "default": true,
+          "description": "integrate flatpak-vscode with other Flatpak extensions like rust-analyzer. Enabling this will make the integrated extensions run their binaries inside the Flatpak sandbox instead of running on the host."
```

Comment:
```suggestion
          "description": "Integrate Flatpak VSCode with other extensions like rust-analyzer. Enabling this will make the integrated extensions run their binaries inside the Flatpak sandbox instead of running on the host."
```